course/templates/course/participation-role-list.html

```diff
@@ -0,0 +1,25 @@
+{% extends "course/course-base.html" %}
+{% load i18n %}
+
+{% load static %}
+
+{% block title %}
+  {% trans "Participation roles" %} - {{ relate_site_name }}
+{% endblock %}
+
+{% block header_extra %}
+  {% include "datatables-header.html" %}
+{% endblock %}
+
+{% block content %}
+  <h1>{% trans "Participation roles" %}</h1>
+
+  <a href="{% url "relate-edit_participation_role" course.identifier -1 %}" class="btn btn-default">{% trans "Add participation role" %}</a>
+
+  {% include "course/participation-role-table.html" with participation_roles=participation_roles %}
+
+{% endblock %}
+
+{% block page_bottom_javascript_extra %}
+    <script src="{{ STATIC_URL }}blueimp-tmpl/js/tmpl.min.js"></script>
+{% endblock %}
```

Comment:
I guess this explains what `tmpl` is. Just now I learned that this got included. I noticed its use in `course/templates/course/grade-flow-page.html`, and I think that code is incredibly gross. I'd rather not see further use of it. I don't know that we need client-side templating.

Reply:
Sorry. I'll check how to remove that.